.expeditor/verify.pipeline.yml

```diff
@@ -780,6 +783,8 @@ steps:
     env:
       HAB_LICENSE: "accept-no-persist"
       BUILD_PKG_TARGET: "x86_64-linux-kernel2"
+      HAB_BLDR_CHANNEL: "stable"
+      HAB_INTERNAL_BLDR_CHANNEL: "stable"
```

Comment:
You want to remove all of these `HAB_INTERNAL_BLDR_CHANNEL` references in this file.